evm/bridge-token-factory/contracts/BridgeTokenFactory.sol

```diff
@@ -28,17 +28,17 @@ contract BridgeTokenFactory is
     address public nearBridgeDerivedAddress;
     uint8 public omniBridgeChainId;
 
-    mapping(uint128 => bool) public completedTransfers;
-    mapping(uint128 => bool) public claimedFee;
-    uint128 public initTransferNonce; 
+    mapping(uint64 => bool) public completedTransfers;
+    mapping(uint64 => bool) public claimedFee;
```

Comment:
Can be removed?

Reply:
Yep, in next [PR](https://github.com/Near-One/omni-bridge/pull/117)